### ISSUE TITLE
training and validation data is the same, but loss and metrics show very different values

### ISSUE THREAD

**Issue by heisenger** [keras-team/keras#13536]
I am using the same training and validation data in a model based on qubvel's segmentation models: 

```
import segmentation_models as sm
sm.FPN(
        backbone,
        classes=classes,
        input_shape=input_shape,
        activation="sigmoid",
        encoder_freeze=true,
    )
```
loss function and metrics are:
`from segmentation_models.losses import bce_jaccard_loss`
`from segmentation_models.metrics import f_score`

However the training and validation scores are very different:
![image](https://user-images.githubusercontent.com/42771859/68529024-f6d8d200-02f1-11ea-9416-1035ff4bd3a8.png)

I have tested it with a simple model (same loss, metric, train and validation data)
`from keras.models import Model`
`from keras.layers import Input, Dense, Conv2D`
`a = Input(shape=(320, 480, 3))`
`b = Conv2D(1, (1, 1))(a)`
`model = Model(inputs=a, outputs=b)`
In this case the train and validation data behave as expected 

![image](https://user-images.githubusercontent.com/42771859/68529051-31426f00-02f2-11ea-8252-5a128f502201.png)

Please can you help me understand why the train and validation metrics can be so different? Does Keras trigger different layers somehow in validation vs train?

Is it correct to understand that the validation and training metrics are shown as averages over the batches?

Thanks!

**Comment by vahe4060** [comment]
Hi. 
I'm facing the same issue, can you please tell me how you fixed it?
